### ISSUE TITLE
[Bug] Execution failure while filtering on datetime column

### ISSUE THREAD

**Issue by saurabh-NECI** [fireducks-dev/fireducks#8]
### Problem
For the below given sample code the execution fails while filtering with 'date' column.

```
import fireducks.pandas as pd

data = {'category': ['A', 'A', 'B', 'B', 'C'],
        'value': [10, 15, 20, 25, 30],
        'date': ['2023-01-01', '2023-01-10', '2022-12-20', '2023-01-15', '2023-01-12']}

df = pd.DataFrame(data)
print(df)

filtered_df = df[df['date'] > '2023-01-10']
print(filtered_df)
```

### Output
```
  category  value       date
0        A     10 2023-01-01
1        A     15 2023-01-10
2        B     20 2022-12-20
3        B     25 2023-01-15
4        C     30 2023-01-12
```

### System details
fireducks version: 0.9.8
OS:                Ubuntu 18.04.6 LTS (GNU/Linux 5.4.0-150-generic x86_64)
Python version:    Python 3.8.18 (default, Nov 30 2023, 05:25:51) [GCC 7.5.0] on linux

### Snippet of error trace (complete trace in attached 'error_trace.txt' file)
```
2024-03-01 06:13:30.342366: 25484 fire/lib/fire_helper.cc:21] Encountered runtime error: NotImplemented: Function 'greater' has no kernel matching input types (timestamp[ns], string)
Traceback (most recent call last):
. . .
  File "/home/demouser/FireDucks/venv/lib/python3.8/site-packages/firefw/runtime.py", line 72, in evaluate
    ret = executor(source, input_values, output_values)
  File "/home/demouser/FireDucks/venv/lib/python3.8/site-packages/fireducks/core.py", line 215, in wrapper
    return fireducks_ext.execute(
fireducks.fireducks_ext.NotImplementedError: Function 'greater' has no kernel matching input types (timestamp[ns], string)
```
[error_trace.txt](https://github.com/fireducks-dev/fireducks/files/14458094/error_trace.txt)

**Comment by qsourav** [comment]
Thank you for reporting the issue.

Please be informed that we have fixed it in our latest released version of FireDucks: 0.10.3.

We would appreciate if you kindly re-evaluate your program with 0.10.3 and confirm the situation.

**Comment by saurabh-NECI** [comment]
The issue no longer exists and its resolution is confirmed successfully in ver. 0.10.3
